pkg/monitor/monitor.go

```diff
@@ -166,11 +164,8 @@ func (monitor *controllerMonitor) getControllerCRD(crdName string) *v1beta1.Antr
 	return controllerCRD
 }
 
-func (monitor *controllerMonitor) createControllerCRD(crdName string) (*v1beta1.AntreaControllerInfo, error) {
-	controllerCRD := &v1beta1.AntreaControllerInfo{
-		ObjectMeta: metav1.ObjectMeta{
-			Name: crdName,
-		},
+func (monitor *controllerMonitor) GetControllerInfo() *v1beta1.AntreaControllerInfo {
+	return &v1beta1.AntreaControllerInfo{
```

Comment:
I think we could add object update time here?

Reply:
I thought about that too but feel the update time might be misleading and does not really provide any extra information, as we do not really cache a resource, but just retrieve the latest state at GET.